Base/usr/share/man/man2/chroot.md

```diff
@@ -17,8 +17,7 @@ int chroot_with_mount_flags(const char* path, int mount_flags);
 
 `chroot_with_mount_flags()` additionally applies the given `mount_flags` to the new root directory
 of the current process as if it was a separate mount. All the same flags as for [`mount`(2)](mount.md)
-are accepted, except `MS_BIND` doesn't make sense for chroots. Additionally, the value -1 is accepted,
-and makes `chroot_with_mount_flags()` behave like regular `chroot()`.
+are accepted, except `MS_BIND`, since it doesn't make sense for chroots. Additionally, the value -1 is accepted, and makes `chroot_with_mount_flags()` behave like regular `chroot()`.
```

Comment:
I believe it is accepted, but doesn't make sense for chroots, exactly as said :slightly_frowning_face:

Reply:
@bugaevc is that a request for a revision, or just a comment on the fact that we accept the flag? :)

Reply:
It's a request for revision, yes.

And TBH I'm not sure I like some other proposed changes; but you should decide on that.

Reply:
Please complain :)

Reply:
Alright :)